cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -21,6 +21,8 @@ predicate localTaintStep(DataFlow::Node nodeFrom, DataFlow::Node nodeTo) {
  */
 predicate localAdditionalTaintStep(DataFlow::Node nodeFrom, DataFlow::Node nodeTo) {
```

Comment:
I think it's about time we make this `cached`.

Reply:
I've started a three-way [CPP-Differences](https://jenkins.internal.semmle.com/job/Changes/job/CPP-Differences/1442/) between main, this branch without caching, and this branch with caching

Reply:
I don't think it'll make a different in our standard suite until `DefaultTaintTracking.qll` is changed to use IR taint-tracking instead of IR data flow.